packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -201,6 +201,11 @@ export interface IInterfaceVpcEndpointService {
    * The port of the service.
    */
   readonly port: number;
+
+  /**
+   * Whether Private DNS is supported by default.
+   */
+  readonly privateDnsDefault?: boolean;
```

Comment:
Why is this optional?

Reply:
I made this optional for backwards compatibility -- any existing CDK code specifying a service like:

```typescript
  const interfaceVpcEndpoint = new ec2.InterfaceVpcEndpoint(stack, 'InterfaceEndpoint', {
    vpc,
    service: {
      name: 'com.amazonaws.us-west-2.workspaces',
      port: 80
    }
  });
```

Will get compilation errors if I don't make the field optional:

```
Property 'privateDnsDefault' is missing in type '{ name: string; port: number; }' 
but required in type 'IInterfaceVpcEndpointService'.
```

Reply:
I see. That's not how you're supposed to supply that value though, but I get it.